Sources/Danger/BitBucketCloud.swift

```diff
@@ -2,227 +2,243 @@ import Foundation
 
-    /// The uuid of the repository
-    public let uuid: String
-}
+        /// The nick name of the user
+        public let nickname: String
 
-public struct BitBucketCloudPRParticipant: Decodable, Equatable {
-    public enum Role: String, Decodable {
-        case reviewer = "REVIEWER"
-        case participant = "PARTICIPANT"
+        /// The uuid of the user
+        public let uuid: String
     }
-
-    /// Did they approve of the PR?
-    public let approved: Bool
-
-    /// How did they contribute
-    public let role: Role
-
-    /// The user who participated in this PR
-    public let user: BitBucketCloudUser
 }
 
-public struct BitBucketCloudUser: Decodable, Equatable {
-    private enum CodingKeys: String, CodingKey {
-        case accountId = "account_id"
-        case displayName = "display_name"
-        case nickname
-        case uuid
-    }
-
-    /// The acount id of the user
-    public let accountId: String
+extension BitBucketCloud {
+    public struct Commit: Decodable, Equatable {
+        public struct Author: Decodable, Equatable {
+            /// Format: `Foo Bar <foo@bar.com>`
+            let raw: String
 
-    /// The display name of user
-    public let displayName: String
+            /// The user that created the commit
+            let user: User
+        }
 
-    /// The nick name of the user
-    public let nickname: String
+        public struct Parent {
+            /// The full SHA
+            let hash: String
+        }
 
-    /// The uuid of the user
-    public let uuid: String
-}
+        /// The author of the commit, assumed to be the person who wrote the code.
+        public let author: Author
 
-public struct BitBucketCloudCommit: Decodable, Equatable {
-    public struct Author: Decodable, Equatable {
-        /// Format: `Foo Bar <foo@bar.com>`
-        let raw: String
+        /// When the commit was commited to the project
+        public let date: Date
 
-        /// The user that created the commit
-        let user: BitBucketCloudUser
-    }
+        /// The SHA for the commit
+        public let hash: String
 
-    public struct Parent {
-        /// The full SHA
-        let hash: String
+        /// The commit's message
+        public let message: String
     }
-
-    /// The author of the commit, assumed to be the person who wrote the code.
-    public let author: Author
-
-    /// When the commit was commited to the project
-    public let date: Date
-
-    /// The SHA for the commit
-    public let hash: String
-
-    /// The commit's message
-    public let message: String
 }
 
-public struct BitBucketCloudComment: Decodable, Equatable {
-    public struct Inline: Decodable, Equatable {
-        public let from: Int
-        public let to: Int // swiftlint:disable:this identifier_name
-        public let path: String
-    }
+extension BitBucketCloud {
+    public struct Comment: Decodable, Equatable {
+        public struct Inline: Decodable, Equatable {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 192;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)